cmd/config/init/init.go

```
@@ -26,6 +28,13 @@ func InitCmd() *cobra.Command {
 			if !validateRollAppID(rollappID) {
 				return fmt.Errorf("invalid RollApp ID '%s'. %s", rollappID, getValidRollappIdMessage())
 			}
+			denom := args[1]
+			if !isValidDenom(denom) {
+				return fmt.Errorf("invalid denom '%s'. %s", denom, validDenomMsg)
```

why not just return the error with the message from the isValidDenom?